### ISSUE TITLE
Design Intent - Design Feedback - Enhanced Military Info

### ISSUE THREAD

**Issue by allison0034** [department-of-veterans-affairs/va.gov-team#72997]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- Thank you for mobile, user flow and current page design!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Must**: Per [our guidance](https://design.va.gov/components/additional-info#when-to-consider-something-else), you have too much content in your additional info component. Link to another page, consider an Accordion, or shorten the content.  IA provided similar feedback: 

>One approach could be to pull some of this info out into a standard heading/paragraph, structure it for scannability, and place it closer to the bottom of the page and the "Feedback" button.

- **Matt Dingee**: It might be good to make the date in the list a date range. I am not sure if that is a start date or what. If I saw something wrong I would want to do something about it. There should be a clear path for me to notify the VA that the info is wrong.

**Comment by andaleliz** [comment]
@allison0034 thank you for the feedback. I hadn't seen that guidance for an additional info component before, and we have many of these components in profile with more content than a single paragraph.

Can accordions be used alone, or should they always be part of a series? I get the sense from the guidance that they're intended to be part of a series.  There aren't any places in profile where a series of accordions would make sense.

Thanks for the clarification!

**Comment by allison0034** [comment]
@andaleliz there is nothing in the guidance against having only 1 accordion. I know I have asked what the difference is between additional info and accordions, visually I know they are different but they behave the same. @briandeconinck mentioned accordions are h2s so that can cause the problem with levels from time to time. @humancompanion-usds there seem to be quit a few instances where designers want to use the additional info component (they do not want to use an h2 for accordion) but cannot because of the [content guidance](https://design.va.gov/components/additional-info#when-to-consider-something-else). Do we think that guidance is still appropriate?

**Comment by humancompanion-usds** [comment]
> Too much content: You should not try to put multiple paragraphs inside Additional info. This includes form fields that require a lot of explanation. Link to another page, consider an [Accordion](https://design.va.gov/components/accordion), or shorten the content.

Multiple != 2. I specifically didn't put a number of paragraphs because I don't want to be pedantic. I think designers know when they have jammed a bunch of content into this component in order to hide it away and not make decisions about what should be included and what shouldn't. The rule exists to ensure that consideration has taken place. I'm not saying this is an instance of that. With that said, how would you recommend I revise this guidance? I'm open to suggestions.

**Comment by andaleliz** [comment]
Thanks for asking that question @allison0034 and @humancompanion-usds, thank you for the clarification! That makes a lot of sense.  I can't speak to other designers, but in my own workflow, I review content (and how it's presented) closely with Danielle and CAIA folks, and I think that process helps to ensure that consideration has taken place. Maybe including something about that could help?

> Too much content: Only include critical information inside Additional Info. This includes form fields that require a lot of explanation. Link to another page, consider an [Accordion](https://design.va.gov/components/accordion), or shorten the content.  Collaborate with a member of our content team to edit content and explore alternatives.

Idk if that's right but it's a start 🙂 

We have _another_ additional info component on this particular page about a similar topic, so ultimately, I think the best thing for this case is to see if we can get a resources and support page created and then link to that.  I'll re-evaluate the content in the other parts of profile as well. 

@humancompanion-usds also want to address your comment on the date range: we don't have an end date available in the data at this point. I'm not even sure if the start date is helpful to include, and we hope to learn about that when we conduct our study in the coming weeks.

**Comment by andaleliz** [comment]
I reviewed the FAQ content with Danielle today, and we decided we're going to create a resources and support page to link to, rather than have 2 additional info components on the page. I submitted a CAIA ticket, #73492, to get this work started.

Since the next phase for us is some generative research to answer the many outstanding questions we have about how we can organize this page for Veterans, and we don't yet have the R&S page created, we're going to hold off on updating the additional info component content in our mock-ups.  

You'll see the updates reflected in the next collab cycle design review :)
